packages/amplify_authenticator/lib/src/utils/validators.dart

```diff
@@ -106,60 +106,76 @@ FormFieldValidator<String> Function(BuildContext) validateNewPassword({
       };
 }
 
-FormFieldValidator<String> validatePasswordConfirmation(
-  String Function() getPassword,
-) {
-  return (String? passwordConfirmation) {
-    if (passwordConfirmation == null || passwordConfirmation.isEmpty) {
-      return 'Re-enter your password to confirm';
-    } else if (getPassword() != passwordConfirmation) {
-      return 'Passwords do not match';
-    }
-    return null;
-  };
+FormFieldValidator<String> Function(BuildContext context)
```

Comment:
Can context be an arg on the first function? The same applies to the other validators.

```dart
FormFieldValidator<String> validatePasswordConfirmation({
  required String Function() getPassword,
  required BuildContext context,
  required InputResolver inputResolver,
}) {
  return (String? passwordConfirmation) {
    if (passwordConfirmation == null || passwordConfirmation.isEmpty) {
      return inputResolver.resolve(
        context,
        InputResolverKey.passwordConfirmationEmpty,
      );
    } else if (getPassword() != passwordConfirmation) {
      return inputResolver.resolve(
          context, InputResolverKey.passwordsDoNotMatch);
    }
    return null;
  };
}
```

consuming it would then look like

```dart
return validatePasswordConfirmation(
  getPassword: () => viewModel.password,
  context: context,
  inputResolver: stringResolver.inputs,
);
```